src/autonlp/autonlp.py

```diff
@@ -101,17 +102,31 @@ def create_project(self, name: str, task: str, language: str, max_models: int):
         self._project.refresh()
         return self._project
 
-    def create_evaluation(self, task: str, dataset: str, model: str, col_mapping: str, split: str, config: str = None):
+    def create_evaluation(
+        self, task: str, dataset: str, model: str, split: str, col_mapping: str = None, config: str = None
```

Comment:
i'm assuming that `col_mapping` is allowed to be `None` in the `payload` - please let me know if this is not allowed

Reply:
not really but we can fix it easily in the backend